public/pages/ReviewAndCreate/components/ModelConfigurationFields/ModelConfigurationFields.tsx

```diff
@@ -188,8 +202,95 @@ export const ModelConfigurationFields = (
     };
   };
 
+  const getFeatureValidationCallout = (
+    issueResponse: ValidationModelResponse
+  ) => {
+    if (issueResponse != undefined) {
+      if (issueResponse.sub_issues != undefined) {
+        const renderList = Object.keys(issueResponse.sub_issues).map((key) => {
+          if (issueResponse.sub_issues != undefined) {
+            return (
+              <li>{'The "' + key + '" ' + issueResponse.sub_issues[key]}</li>
+            );
+          }
+        });
+        return <ul>{renderList}</ul>;
+      } else {
+        return (
+          <ul>
+            <li>{JSON.stringify(props.validationFeatureResponse.message)}</li>
+          </ul>
+        );
+      }
+    } else {
+      return null;
+    }
+  };
+  const getValidationCallout = () => {
+    //When validation response is loading then displaying loading spinner, don't display
+    // after clicking on "create detector" button as isLoading will be true from that request
+    if (props.isLoading && !props.isCreatingDetector) {
+      return (
+        <EuiCallOut
+          title={
+            <div>
+              <EuiFlexGroup direction="row" gutterSize="xs">
+                <EuiLoadingSpinner size="l" style={{ marginRight: '12px' }} />
+                <EuiText>
+                  <p>Validating model configurations</p>
+                </EuiText>
+              </EuiFlexGroup>
+            </div>
+          }
+          style={{ marginBottom: '10px' }}
+          size="s"
+          color="primary"
+        />
+      );
+    }
+    if (props.validationError) {
+      return null;
+    } else if (props.validModel) {
```

Comment:
If user `return`, how about we simply use `if` block?

Reply:
sure, changed this